packages/lab/.gitignore

```diff
@@ -0,0 +1 @@
+../../.gitignore
```

Comment:
Do we need this file?

Reply:
Apparently not! (Can't repro whatever it was that required having a package local `.gitignore`...)